addons/account/models/account_move.py

```diff
@@ -581,6 +581,7 @@ def _sequence_fixed_regex(self):
     )
     partner_credit = fields.Monetary(compute='_compute_partner_credit')
     duplicated_ref_ids = fields.Many2many(comodel_name='account.move', compute='_compute_duplicated_ref_ids')
```

Comment:
Question: I've had to make a new field instead of using the duplicated_ref_ids because adding the change like here https://github.com/odoo/odoo/commit/18fbba03aee0a7e5562f33acaf426101b94b5238 seems to break a lot of queries due to newId stuff, maybe we can check together on monday what is the best way to do this @Levizar

Reply:
Also still some doubt about using a search and not an sql query

Reply:
Found a way to use the old methold and field duplicated_ref, the other ways didn't worked because of newIDs